modules/dag-l1/src/main/scala/org/tessellation/dag/l1/package.scala

```diff
@@ -29,6 +29,11 @@ package object l1 {
     classOf[ConsensusInput.Proposal] -> 810,
     classOf[ConsensusInput.SignatureProposal] -> 811,
     classOf[ConsensusInput.CancelledCreationRound] -> 812,
-    DataCancellationReason.getClass -> 813
+    DataCancellationReason.getClass -> 813,
+    DataCancellationReason.ReceivedProposalForNonExistentOwnRound.getClass -> 814,
+    DataCancellationReason.MissingRoundPeers.getClass -> 815,
+    DataCancellationReason.CreatedInvalidBlock.getClass -> 816,
+    DataCancellationReason.CreatedEmptyBlock.getClass -> 817,
+    DataCancellationReason.PeerCancelled.getClass -> 818
```

Comment:
@TheMMaciek Makes sense. I understand why @IPadawans put them there though: `DataCancellationReason.getClass` -- are we allowed to move entries once they've been in production or are they here forever?

Reply:
I suspect it depends if the use of those values affects things beyond the life of the running program because moving it means we should change the value to match the range in its new location.

Reply:
I've updated the PR according to what we discussed

Reply:
Maybe because it's an error class it's safe to move/change the value but I'm tempted to leave that one where it is and just add a comment saying why we left it.

Reply:
IMO if we use the `sharedKryoRegistrar` we can move the `DataCancellationReason.getClass` but also let register with a comment with the older ID on the other file